generate_ids.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Maybe this should be documented somewhere? This will need to be run on each environment we deploy